common/rfb/Logger.cxx

```diff
@@ -37,6 +37,9 @@ Logger::~Logger() {
   // *** Should remove this logger here!
 }
 
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wformat-nonliteral"
```

Comment:
Rather than simply silencing this, I think we should try to help clang out. Apparently it wants a `__printf__` attribute for these cases:

https://clang.llvm.org/docs/AttributeReference.html#id149

See `LogWriter.h` for something similar.